### PR TITLE
update monitoring.md with different status types

### PR DESCRIPTION
**Pull request by susanev**
Signed-off-by: susanev <susan.ra.evans@gmail.com>

### :nut_and_bolt: Description: What code changed, and why?
update monitoring.md with different status types
see [internal slack convo](https://chefio.slack.com/archives/CARK1CBNX/p1598021894034800)

### :chains: Related Resources
zero

### :+1: Definition of Done
docs have more info

### :athletic_shoe: How to Build and Test the Change
use the netlify link

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [x] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: components/docs-chef-io/content/automate/monitoring.md [hunk 1]
```diff
@@ -15,12 +15,12 @@ draft = false
 
 ## Overview
 
-The `/status` endpoint is an authenticated API endpoint for users who want to monitor their Automate installation by connecting to an http endpoint. Previously, users ran the `chef-automate status` command from the Automate node itself to monitor an Automate installation.
```
[the remark this answers is not in the document]
Comment (kagarmoe) [reply components/docs-chef-io/content/automate/monitoring.md:18]: The remaining sentence is really circular, in effect, we're saying that the `/status` endpoint is for people who want to use an endpoint. I'm sure it made sense when we wrote it, but in what circumstances does someone want to use an HTTP connection?

[the remark this answers is not in the document]
Comment (susanev) [reply components/docs-chef-io/content/automate/monitoring.md:18]: we could link to the cli section if we wanted to?

[the remark this answers is not in the document]
Comment (yzl) [reply components/docs-chef-io/content/automate/monitoring.md:18]: @susanev @kagarmoe In the context of monitoring, I wouldn't mention the CLI at all, on the grounds that `/status` is for consumption by monitoring systems (e.g. nagios, sensu, datadog) while `chef-automate status` is for consumption by people. I would probably say something like "Use the authenticated https endpoint`/status` to monitor your Chef Automate installation" or "Chef Automate provides the authenticated HTTPS endpoint `/status` for use by monitoring systems."

File: components/docs-chef-io/content/automate/monitoring.md [hunk 2]
```diff
@@ -13,53 +13,76 @@ draft = false
...
+             "name": "automate-postgresql",
+             "status": "OK"
+           },
+         ]
+       }
+   ```
+
+   If access is not granted, then the output indicates this:
+
+   ```json
```
Comment (mjingle) [inline components/docs-chef-io/content/automate/monitoring.md:68]: @yzl I appreciate you providing that perfectly not-perfect example in your comment! Does this incorporate your example as desired?

Comment (susanev) [reply components/docs-chef-io/content/automate/monitoring.md:68]: wait, what does it mean `if access is not granted?`

Comment (mjingle) [reply components/docs-chef-io/content/automate/monitoring.md:68]: Was basing phrasing on [line 38](https://github.com/chef/automate/pull/4259/files#diff-cd26af5d6bb3b04cb8b46e56dce74c3cR38) - "Test that your token and policy give you access to the `/status` endpoint" - but I'm not entirely satisfied with what I put on line 66.

Comment (yzl) [reply components/docs-chef-io/content/automate/monitoring.md:68]: @mjingle Not quite. The point I meant to make with the example is that the JSON output for a service will look like

```
{
             "name": "<service-name>",
             "status": "<service-status>"
           },
```

where service-status can be OK (service is up and healthy), CRITICAL (service is responding but not healthy), or UNKNOWN (service is down/unloaded).

Comment (mjingle) [reply components/docs-chef-io/content/automate/monitoring.md:68]: D'oh! I am following now. Thank you for clarifying @yzl. Let me see if I can change it accordingly.

Comment (mjingle) [reply components/docs-chef-io/content/automate/monitoring.md]: @yzl Changed!

File: components/docs-chef-io/content/automate/monitoring.md [hunk 3]
```diff
@@ -13,16 +13,37 @@ draft = false
...
 [\[edit on GitHub\]](https://github.com/chef/automate/blob/master/components/docs-chef-io/content/automate/monitoring.md)
 
-## Overview
+Use the authenticated https endpoint `/status` to monitor your Chef Automate installation.
 
-The `/status` endpoint is an authenticated API endpoint for users who want to monitor their Automate installation by connecting to an http endpoint. Previously, users ran the `chef-automate status` command from the Automate node itself to monitor an Automate installation.
+## Checking the Status Endpoint
 
-## Checking the status endpoint
+The authenticated endpoint `/status` provides status for the overall Chef Automate installation as well as its component services.
```
Comment (yzl) [inline components/docs-chef-io/content/automate/monitoring.md:20]:
```suggestion
The authenticated endpoint `/status` provides status for the overall Chef Automate installation as well as for its component services.
```

**Pull request merged**